cmd/sops/subcommand/updatekeys/updatekeys.go

```diff
@@ -66,11 +66,22 @@ func updateFile(opts Opts) error {
 			keysWillChange = true
 		}
 	}
-	if !keysWillChange {
+
+	// TODO: use conf.ShamirThreshold instead of tree.Metadata.ShamirThreshold in the next line?
+	//       Or make this configurable?
+	var shamirThreshold = tree.Metadata.ShamirThreshold
+	if opts.GroupQuorum != 0 {
```

Comment:
The term `GroupQuorum` is only used here. The term `GroupThreshold` is used in options in the `groups add` and `groups delete `commands, while `ShamirThreshold` is used elsewhere. It is not terribly important, but renaming the field in the `Opts` struct would improve readability imho.

Reply:
I agree, but that should happen in a feature (or even major) release, not in a bugfix release. While this is not part of the offically declared public interface, we still try to avoid breaking changes there.

Reply:
> I agree, but that should happen in a feature (or even major) release, not in a bugfix release. While this is not part of the offically declared public interface, we still try to avoid breaking changes there.

Excellent point 👍